README.md

```diff
@@ -57,9 +64,9 @@ const createSlackEventAdapter = require('@slack/events-api').createSlackEventAda
 const slackEvents = createSlackEventAdapter(process.env.SLACK_VERIFICATION_TOKEN);
 const port = process.env.PORT || 3000;
 
-// Attach listeners to events by Slack Event. See: https://api.slack.com/events/api
-slackEvents.on('message.im', (event)=> {
-  console.log(`Received a DM event: user ${event.user} in channel ${event.channel} says ${event.text}`);
+// Attach listeners to events by Slack Event "type". See: https://api.slack.com/events/message.im
```

Comment:
Yes, it's not the page with all the events, it's the page with a specific event which has "message" as it's "type". I thought this would be clearer to illustrate what I meant by a Slack Event "type". Does that work?